src/OrbitBase/include/OrbitBase/Typedef.h

```diff
@@ -70,10 +70,12 @@ class Typedef {
   constexpr Typedef(Typedef&& other) = default;
 
   template <typename U, typename = orbit_base_internal::EnableIfUConvertibleToT<T, U>>
-  constexpr Typedef(const Typedef<Tag_, U>& other) : value_(*other) {}
```

Comment:
@akopich, it would be nice if we could have the constructor explicit iff the wrapped type's constructor is explicit.
Could you have a look?

Reply:
Thanks for the clarification.